go/analyzer/staticcheck/BUILD.bazel

```diff
@@ -0,0 +1,185 @@
+# gazelle:ignore
```

Comment:
Is this needed? If it is, could we prefer # keep instead of ignoring entire directories.